docs/website/docs/general-usage/http/rest-client.md

```diff
@@ -518,6 +565,74 @@ client.paginate("/posts", hooks={"response": [custom_response_handler]})
 
 The handler function may raise `IgnoreResponseException` to exit the pagination loop early. This is useful for the enpoints that return a 404 status code when there are no items to paginate.
 
+### Incremental loading
+
+It is often needed to load only the new data based on some incremental property be it timestamp, date and time, integer identifier or a cursor value.
+Fortunately our `RESTClient` allows you to elegantly express this behavior.
+
+Let's use our slightly modified example response json and we want to load new posts as they appear without complete reload of data.
+
+```json
+{
+  "data": [
+    { "id": 1, "title": "Post 1", "created_at": "2010-08-21T17:11:27-0400" },
+    { "id": 2, "title": "Post 2", "created_at": "2010-09-21T17:11:27-0400" },
+    { "id": 3, "title": "Post 3", "created_at": "2010-10-21T17:11:27-0400" }
+  ]
+}
+```
+
+To achive our objective we need to use `endpoint.params` by adding the incremental type.
+In the following examples we use `id` - primary key and `created_at` - creation datetime.
+
+**Incremental loading by id**
+
+```py
+source_config: RESTAPIConfig = {
+    "resources": [
+        {
+            "name": "get_posts_list",
+            "table_name": "posts",
+            "endpoint": {
+                "data_selector": "$.data",
+                "path": "/posts",
+                "params": {
+                    "post_id": {
+                        "type": "incremental",
+                        "cursor_path": "id",
+                        "initial_value": 1,
+                    }
+                },
+            },
+        }
+    ]
+}
+```
+
+**Incremental loading by creation date**
+
+```py
+source_config: RESTAPIConfig = {
+    "resources": [
+        {
+            "name": "get_posts_list",
+            "table_name": "posts",
+            "endpoint": {
+                "data_selector": "$.data",
+                "path": "/posts",
+                "params": {
+                    "creation_date": {
+                        "type": "incremental",
+                        "cursor_path": "created_at",
+                        "initial_value": "2010-08-21T17:11:27-0400",
+                    }
+                },
+            },
+        }
+    ]
+}
+```
+
```

Comment:
This is not the right place for this section: rest-client.md is only for documenting RESTClient class & relevant functionality and not rest_api source. Incremental loading is covered in rest_api here: https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api#incremental-loading